OpenAndReadsCSVInWindow

```diff
@@ -14,8 +14,15 @@ class CourseInfoApp:
         self.root.iconphoto(False, icon)
 
         
-        self.browse_button = tk.Button(root, text="Browse", command=self.open_file_dialog)
```

Comment:
Honestly don't use GitHub much and I saw two lines 17-18 (red) 17-18 (green). I'm assuming that was just you editing code and I thought that was two different buttons you made.. sorry for the confusion!